css/style.css

```diff
@@ -0,0 +1,201 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body{
+    font-size: 15px;
```

Comment:
Is a font size needed?

Reply:
think its used to set 1 rem so i can use 1.5rems etc later